crates/cheatcodes/src/inspector.rs

```diff
@@ -833,6 +833,19 @@ impl Cheatcodes {
 
         // Apply our prank
         if let Some(prank) = &self.prank {
+            // Apply delegate call. call.caller will not equal prank.prank_caller
```

Comment:
Hey @zerosnacks, my understanding is a little fuzzy here. It seems in the case of a delegate call that `call.caller == prank.prank_caller` will not be true. Is it suitable to have this here? Ty.

Reply:
cc @DaniPopes would you have more context on this?